### DISCUSSION TITLE
Problema com versões mais antigas do IE e blocos

### DISCUSSION THREAD

**Discussion by iAlfredoMiranda** [frontendbr/forum#1399] [Perguntas e Respostas]
Olá, Alguém poderia me ajudar. Estou com problemas em versões do IE11 11.0.9 e inferiores. 
O meu menu não está se formando direito. Era para ter uma imagem de banner se encaixando por de baixo do menu transparente. Mas isso não acontece. A página carrega e o joga a imagem para baixo como se fossem separados. Alguém sabe o que poderia ser?

**Comment by luizbills** [comment]
Posta o HTML e o CSS para ver seu problema acontecer.
